OpenDungeons/OpenDungeons
# Add console command to add gold to players and support to add gold in a ...

...level file.

If treasuries are full, remaining gold is dropped
fixes #117

Added map "Stonekeep siege.level" from @Danimal696 

To test it :
- Check  that the gold amount in a level file is added to the treasuries
- In the console, use addgold command to add gold in you treasuries
